server/queues/tasks/ExportDocumentTreeTask.ts

```diff
@@ -73,14 +74,17 @@ export default abstract class ExportDocumentTreeTask extends ExportTask {
             date: attachment.updatedAt,
             createFolders: true,
           });
+
+          text = text.replace(
```

Comment:
Fix a bug where only the first instance of an attachment would be replaced in the output, subsequent identical images would not be.